Examples/test-suite/python/langobj_runme.py

```diff
@@ -3,6 +3,10 @@
 
 
 x = 256*256+1
+
+# avoid immortal object in no-gil scenario
+x += 1
+
```

Comment:
Could you elaborate on this please? I don't understand why immortal objects are not allowed in no-gil scenarios.